pkg/cmd/taskrun/logs.go

```diff
@@ -0,0 +1,110 @@
+package taskrun
+
+import (
+	"fmt"
+	"github.com/tektoncd/cli/pkg/cli"
```

Comment:
nit: needs an empty line :angel:

Reply:
resolved, please feel free to re-open it :sweat_smile: